pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -214,6 +217,28 @@ self: super: {
     dependencies = with super; [ super.self ];
   });
 
+  fruzzy = let # until https://github.com/NixOS/nixpkgs/pull/67878 is merged, there's no better way to install nim libraries with nix
+    nimpy = fetchTarball "https://github.com/yglukhov/nimpy/archive/master.tar.gz";
+    binaryheap = fetchTarball "https://github.com/bluenote10/nim-heap/archive/master.tar.gz";
```

Comment:
this only works locally, you need to use `fetchurl` + sha256

```
access to URI 'https://github.com/yglukhov/nimpy/archive/master.tar.gz' is forbidden in restricted mode
```

Reply:
I fixed it by providing sha256 sums as you suggested